portfolio/src/main/webapp/index.html

```diff
@@ -12,6 +12,7 @@
   </head>
   <body>
     <div class="text-center font"> 
+        <a href="projects.html"> Projects </a>
```

Comment:
nit: 2 spaces for indentation (if tab characters are defaulting to 4 spaces, as they often do in some editors, there should be a setting to change that)

Reply:
fixed, will have to look into changing my tabs